apps/i18n/javalab/en_us.json

```diff
@@ -156,6 +157,7 @@
   "unsupportedMiniAppMessage": "Level type mismatch: {miniAppType} projects are not supported on a non-{miniAppType} level.",
   "userBlocked": "You have run your code too many times and your account has been locked. Contact support@code.org to unlock your account.",
   "userBlockedTemporary": "You have reached the limit of the number of times you can run Java code in 24 hours. Become a verified teacher to access a much higher quota for yourself and your students.\n You can learn more about becoming verified [here](https://support.code.org/hc/en-us/articles/115001550131-How-to-Become-a-Verified-Teacher).\n You can learn why verification is needed [here](https://support.code.org/hc/en-us/articles/6104078305549-Teacher-Verification-and-CSA).\n If you believe this message is in error, contact support@code.org.",
+  "waitingForServer": "Your request has been put in line. Please hold as we wait for the next available machine.",
```

Comment:
Thinking about the production javabuilder & verified user case.

Is there any strange state I could get in where I'd see this as a verified user (perhaps if we're in a scale-up period or if the user has a really slow internet connection or if some message gets dropped)? I'm wondering because it seems like it would be confusing for a verified user to see this. Trying to think through whether that would be possible.

Reply:
I think it would be possible in either the slow network or scale-up scenario. We could update it to be something more generic like "waiting for a response" if that would be clearer?

Reply:
[discussed in slack](https://codedotorg.slack.com/archives/C03DBDN67B7/p1683048364741549) and updated to "Waiting for a response..."